pyproject.toml

```diff
@@ -54,6 +54,7 @@ click = { version = "^8.1.3", optional = true }
 rich = { version = "^13.4.1", optional = true }
 cookiecutter = { version = "^2.1.1", optional = true }
 jinja2-time = { version = "^0.2.0", optional = true }
+pydispatcher = "^2.0.7"
```

Comment:
seems like  it wasn't done using poetry, we need to have the .lock file modified as well

Reply:
there is obe